src/app/zap-templates/zcl/data-model/chip/camera-av-stream-management-cluster.xml

```diff
@@ -0,0 +1,578 @@
+      <access op="write" privilege="administer"/>
+    </attribute>
+    <attribute code="0x001E" side="server" define="ISO" type="boolean" writable="true" optional="true">
+      <description>ISO</description>
+      <access op="read" privilege="administer"/>
+      <access op="write" privilege="administer"/>
+    </attribute>
+    <attribute code="0x001F" side="server" define="VIEWPORT" type="ViewportStruct" optional="true">
+      <description>Viewport</description>
+      <access op="read" privilege="administer"/>
+    </attribute>
+    <attribute code="0x0020" side="server" define="SPKR_ON_OFF" type="boolean" writable="true" optional="true">
+      <description>SpkrOnOff</description>
+      <access op="read" privilege="administer"/>
+      <access op="write" privilege="administer"/>
+    </attribute>
+    <attribute code="0x0021" side="server" define="SPKR_VOLUME_LEVEL" type="int8u" max="254" writable="true" optional="true">
+      <description>SpkrVolumeLevel</description>
+      <access op="read" privilege="administer"/>
+      <access op="write" privilege="administer"/>
+    </attribute>
+    <attribute code="0x0022" side="server" define="SPKR_MAX_LEVEL" type="int8u" max="254" writable="true" optional="true">
+      <description>SpkrMaxLevel</description>
+      <access op="read" privilege="administer"/>
+      <access op="write" privilege="administer"/>
+    </attribute>
+    <attribute code="0x0023" side="server" define="SPKR_MIN_LEVEL" type="int8u" max="254" writable="true" optional="true">
+      <description>SpkrMinLevel</description>
+      <access op="read" privilege="administer"/>
+      <access op="write" privilege="administer"/>
+    </attribute>
+    <attribute code="0x0024" side="server" define="MIC_ON_OFF" type="boolean" writable="true" optional="true">
+      <description>MicOnOff</description>
+      <access op="read" privilege="administer"/>
+      <access op="write" privilege="administer"/>
+    </attribute>
+    <attribute code="0x0025" side="server" define="MIC_CURRENT_LEVEL" type="int8u" max="254" writable="true" optional="true">
+      <description>MicCurrentLevel</description>
+      <access op="read" privilege="administer"/>
+      <access op="write" privilege="administer"/>
+    </attribute>
+    <attribute code="0x0026" side="server" define="MIC_MAX_LEVEL" type="int8u" max="254" writable="true" optional="true">
+      <description>MicMaxLevel</description>
+      <access op="read" privilege="administer"/>
+      <access op="write" privilege="administer"/>
+    </attribute>
+    <attribute code="0x0027" side="server" define="MIC_MIN_LEVEL" type="int8u" max="254" writable="true" optional="true">
+      <description>MicMinLevel</description>
+      <access op="read" privilege="administer"/>
+      <access op="write" privilege="administer"/>
+    </attribute>
+    <attribute code="0x0028" side="server" define="MIC_AGC" type="boolean" writable="true" optional="true">
+      <description>MicAGC</description>
```

Comment:
Spec comment: Consider calling this MicAGCEnabled for clarity.

Reply:
Done.